dt-impact-tracer/src/edu/washington/cs/dt/impact/Main/ResultsParser.java

```diff
@@ -3,15 +3,150 @@
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
+import java.util.Scanner;
+import java.io.*;
```

Comment:
Instead of importing the entire package, you should be able to import only the classes you need.
Ctrl + Shift + O  in Eclipse should automatically fix this for you.